app/containers/ItunesContainer/reducer.js

```diff
@@ -4,22 +4,36 @@
  *
  */
 import produce from 'immer';
+import { get } from 'lodash';
 import { createActions } from 'reduxsauce';
 
-export const initialState = {};
+export const initialState = { ituneName: null, itunesData: {}, itunesError: null };
```

Comment:
won't we need some loading state as well ?

Reply:
Loading state will not be in redux store but it will be in a Itunes Container file which will be added in next PR.